src/wp-content/themes/twentynineteen/inc/template-functions.php

```diff
@@ -35,12 +35,12 @@ function twentynineteen_body_classes( $classes ) {
 /**
  * Adds custom class to the array of posts classes.
  */
-function twentynineteen_post_classes( $classes, $css_class, $post_id ) {
+function twentynineteen_post_classes( $classes ) {
```

Comment:
Since touching this function, maybe consider adding the `@param` to the function's docblock.

Reply:
The feedback addressed in https://github.com/WordPress/wordpress-develop/commit/dec4b321f99acf4c62dfe2d9f7e2513ebeaf26e3